### PR TITLE
feat: main versions misalignment initial

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: examples/scheduled-actions-workspace/scheduled.code-workspace [hunk 1]
```diff
@@ -8,13 +8,5 @@
     }
   ],
   "settings": {
-    "actions": [
```
Comment (bd82) [inline examples/scheduled-actions-workspace/scheduled.code-workspace:11]: should this have been deleted?

File: packages/npm-dependencies-validation/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@sap-devx/npm-dependencies-validation",
+  "version": "1.5.0",
+  "private": true,
```
Comment (bd82) [inline packages/npm-dependencies-validation/package.json:4]: should this remain private?

File: packages/vscode-dependencies-validation/.mocharc.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const baseConfig = require("../../.mocharc.js");
+
+module.exports = {
+  ...baseConfig,
+  spec: "./dist/test/**/*spec.js",
+  // TODO: large timeout only for specific tests
```
Comment (bd82) [inline packages/vscode-dependencies-validation/.mocharc.js:6]: i will write a note to fix this later

File: packages/vscode-dependencies-validation/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# vscode-dependencies-validation README
```
Comment (bd82) [inline packages/vscode-dependencies-validation/README.md:1]: we need content in this README
We may also want an example demonstrating the functionality.

The README is more urgent

File: packages/vscode-dependencies-validation/src/diagnostics/debounce.ts [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+//
+// So a large interval is used to reduce filesystem load from this cleanup process.
+const GC_INTERVAL = 15 * MINUTE;
+
+/* istanbul ignore next -- TCO of testing this setInterval is too high*/
+setInterval(() => {
+  try {
+    garbageCollect(workspace.getWorkspaceFolder);
+  } catch (e) {
+    // TODO: log this error once logging is implemented
```
Comment (bd82) [inline packages/vscode-dependencies-validation/src/diagnostics/debounce.ts:46]: missing log

File: packages/vscode-dependencies-validation/test/vscodeMocks.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import { NPM_DEPENDENCY_ISSUES_CODE } from "../src/constants";
```
Comment (bd82) [inline packages/vscode-dependencies-validation/test/vscodeMocks.ts:1]: is this file used?
it does not seem to be rendered as TypeScript by github for some reason

File: packages/vscode-dependencies-validation/webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+const path = require("path");
```
Comment (bd82) [inline packages/vscode-dependencies-validation/webpack.config.js:1]: also strange, this file does not seem to be redered as JS

**Review by bd82 — APPROVED** [review]
see inline

File: examples/scheduled-actions-workspace/scheduled.code-workspace (hunk 1 above)
Comment (slavik-lvovsky) [reply examples/scheduled-actions-workspace/scheduled.code-workspace]: Done

File: packages/vscode-dependencies-validation/src/diagnostics/debounce.ts (hunk 5 above)
Comment (slavik-lvovsky) [reply packages/vscode-dependencies-validation/src/diagnostics/debounce.ts]: Done

**Pull request merged**